prvocisla.cpp

```diff
@@ -1,14 +1,14 @@
 #include <iostream>
-
 #include <string.h>
 
-
 using namespace std;
 
 int * sieve(int n){
     int * A;
     int * L;
+    
     int numPrime;
+    
     A =(int *) malloc(n*sizeof(int));
     memset(A,0,8);
```

Comment:
ja myslim ze tomu musis dat `n`

Reply:
cemu presne

Reply:
no asi ten radek ke kteremu to pisu - memset ... mas tam 8; takze nastavis na 0 jen prvnich 8 bytu.

Myslim ze takto:
```suggestion
    memset(A,0,n*sizeof(int));
```
https://www.cplusplus.com/reference/cstring/memset/
https://www.cplusplus.com/forum/articles/416/

Reply:
> cemu presne

tomu rikam "velice konkretni" otazka ... nemas ani otaznik lol

Reply:
oukej. ja jsem te blbe pochopila.